<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="3" passed="1" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-05-08T15:02:36 IST" name="Default suite" finished-at="2022-05-08T15:02:45 IST" duration-ms="8975">
    <groups>
    </groups>
    <test started-at="2022-05-08T15:02:36 IST" name="Default test" finished-at="2022-05-08T15:02:45 IST" duration-ms="8975">
      <class name="Test.logintest">
        <test-method is-config="true" signature="addtestreport()[pri:0, instance:Test.logintest@273e7444]" started-at="2022-05-08T15:02:36 IST" name="addtestreport" finished-at="2022-05-08T15:02:36 IST" duration-ms="317" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addtestreport -->
        <test-method is-config="true" signature="launchBrowser()[pri:0, instance:Test.logintest@273e7444]" started-at="2022-05-08T15:02:36 IST" name="launchBrowser" finished-at="2022-05-08T15:02:45 IST" duration-ms="8296" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method signature="checkforgot()[pri:0, instance:Test.logintest@273e7444]" started-at="2022-05-08T15:02:45 IST" name="checkforgot" finished-at="2022-05-08T15:02:45 IST" duration-ms="292" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkforgot -->
        <test-method is-config="true" signature="captureresult(org.testng.ITestResult)[pri:0, instance:Test.logintest@273e7444]" started-at="2022-05-08T15:02:45 IST" name="captureresult" finished-at="2022-05-08T15:02:45 IST" duration-ms="6" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=checkforgot status=SUCCESS method=logintest.checkforgot()[pri:0, instance:Test.logintest@273e7444] output={null}]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because "this.test" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because "this.test" is null
at Test.logintest.captureresult(logintest.java:102)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:65)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:381)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:319)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:803)
at org.testng.internal.invokers.TestInvoker.runAfterConfigurations(TestInvoker.java:772)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:748)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- captureresult -->
        <test-method is-config="true" signature="flushresult()[pri:0, instance:Test.logintest@273e7444]" started-at="2022-05-08T15:02:45 IST" name="flushresult" finished-at="2022-05-08T15:02:45 IST" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flushresult -->
      </class> <!-- Test.logintest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
